workflows/phylogenetics/wf_nextclade_addToRefTree.wdl

```diff
@@ -8,28 +8,22 @@ workflow nextclade_addToRefTree {
       description: "Nextclade workflow that adds samples to a curated JSON tree from Augur."
     }
     input {
-      File assembly_fastas
-      File? root_sequence_fasta
+      File assembly_fasta
       File? gene_annotations_gff
       File? reference_tree_json
-      File? qc_config_json
+      File? nextclade_pathogen_json
       File? pcr_primers_csv
-      File? virus_properties
       String nextclade_dataset_name
-      String? dataset_reference
       String? dataset_tag
     }
     call nextclade_analysis.nextclade_add_ref { # nextclade analysis
       input:
-        genome_fasta = assembly_fastas,
-        root_sequence = root_sequence_fasta,
+        genome_fasta = assembly_fasta,
         reference_tree_json = reference_tree_json,
-        qc_config_json = qc_config_json,
+        nextclade_pathogen_json = nextclade_pathogen_json,
         gene_annotations_gff = gene_annotations_gff,
         pcr_primers_csv = pcr_primers_csv,
```

Comment:
Same comment above RE input removed

Reply:
resolved in [0800fa0](https://github.com/theiagen/public_health_bioinformatics/pull/375/commits/0800fa0ce76b0faead325227b1ec2cda428361c9)